README.md

```diff
@@ -67,7 +66,7 @@ yarn add \
   @near-wallet-selector/near-snap \
   @near-wallet-selector/ledger \
   @near-wallet-selector/wallet-connect \
-  @near-wallet-selector/nightly-connect \
+  @near-wallet-selector/default-wallets \
```

Comment:
This line should be removed, `default-wallets` package has been deprecated a few months ago:
https://www.npmjs.com/package/@near-wallet-selector/default-wallets